test/api/constructor/index.ts

```diff
@@ -1,29 +0,0 @@
-import assert from 'assert-diff'
-import {TestSuite} from '../../utils'
-import {RippleAPI} from 'ripple-api'
```

Comment:
These files were just moved to `test/client`, not deleted - I think git had a hard time with the diff